srv/salt/ceph/ganesha/restart/default.sls

```diff
@@ -0,0 +1,20 @@
+restart:
+  cmd.run:
```

Comment:
Sorry, I'm out of the loop a little bit...is there a reason for not using the service module? As in service.restart?